docs/source/operations/besu_networkyaml.md

```diff
@@ -344,5 +344,7 @@ The fields under `validator` service are
 | rpc.port   | RPC port for Besu|
 | rpc.ambassador | The RPC Port when exposed on ambassador service|
 | ws.port   | Webservice port for Besu|
+| metrics.enabled | Enable metrics for Besu node |
```

Comment:
I don't think we need the `enabled` field. This could be easily considered by the presence/absence of the metrics port.